clang/include/clang/Basic/LangOptions.h

```diff
@@ -131,6 +131,9 @@ class LangOptions : public LangOptionsBase {
     SYCL_None,
```

Comment:
Do we need SYCL_None now?

Reply:
Yes -- that's for the case where the user doesn't pass `-fsycl-is-device` or `-fsycl-is-host`.